packages/browser-wallet/src/wallet-api/compatibility.ts

```diff
@@ -34,7 +34,7 @@ import {
     getAccountTransactionHandler,
     AccountTransactionPayloadJSON,
 } from '@concordium/web-sdk/types';
-import { empty } from '@concordium/web-sdk/types/Parameter';
+import { Parameter } from '@concordium/web-sdk';
```

Comment:
I think this change will include the wasm module again, since esbuild tree-shaking is not smart enough

Reply:
ah fuck.. good catch! I'll see if I can change it back.. changed it due to build errors